src/button_handler.cpp

```diff
@@ -209,6 +256,14 @@ void Handler::resetPressed(sdbusplus::message::message& msg)
 {
     try
     {
+        if (!isBmcReady())
```

Comment:
What is a "reset" button? afaik, the pinhole reset is routed directly to the bootfail chip and not to this app.

Again. this one too seems to be applicable on when we are already powered ON, so the BMC state should not matter.

Reply:
OK, you are saying it's a cold reset, and the the host is already powered ON so no need to check BMC state. Makes sense, Will remove this check.

Maybe @spinler can help me/us on the purpose of the "reset" button - just for our understanding :)